vm-setup/roles/v1aX_integration_test/tasks/remediation.yml

```diff
@@ -89,23 +106,25 @@
   - name: Power cycle a single worker node
     include: power_cycle.yml
     vars:
-      NODE: '{{ WORKER_NODE }}'
+      NODE: '{{ WORKER_BMH }}'
       NODE_VM: "{{ WORKER_NODE_VM }}"
+      NODE_NAME: "{{ WORKER_NODE }}"
```

Comment:
These names are a bit confusing. Could you please rename them to be more descriptive? 
example:
```
      BMH_NODE: '{{ WORKER_BMH }}'
      LIBVIRT_VM: "{{ WORKER_NODE_VM }}"
      K8S_NODE:  "{{ WORKER_NODE }}"
```

Reply:
This is not resolved.